src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/ContentItemsFieldType.cs

```diff
@@ -109,14 +111,14 @@ private async Task<IEnumerable<ContentItem>> Resolve(ResolveFieldContext context
             // Add all provided table alias to the current predicate query
             var providers = context.ServiceProvider.GetServices<IIndexAliasProvider>();
             var indexes = new Dictionary<string, IndexAlias>(StringComparer.OrdinalIgnoreCase);
-            var indexAliases = new List<string>();
+            var indexAliases = new Dictionary<string, string>();
 
             foreach (var aliasProvider in providers)
             {
                 foreach (var alias in aliasProvider.GetAliases())
                 {
                     predicateQuery.CreateAlias(alias.Alias, alias.Index);
-                    indexAliases.Add(alias.Alias);
+                    indexAliases.Add(alias.Alias.ToLower(), alias.Alias);
```

Comment:
Prefer a case insensitive dictionary

Reply:
@sebastienros me too not sure what came over me ..

Reply:
@sebastienros resolved